include/odp/api/spec/pool.h

```diff
@@ -518,6 +518,49 @@ unsigned int odp_pool_max_index(void);
  */
 int odp_pool_index(odp_pool_t pool);
 
+/**
+ * Pool statistics
+ */
+typedef struct odp_pool_stats_t {
+	/**
+	 * The number of available entries in the pool.
+	 * When the local cache is enabled, the count will account for the local cache
+	 * entries per CPU as well.
+	 */
+	uint64_t available;
```

Comment:
Feels that only counting free or allocated events is enough, application can calculate the other figure. Maybe separate counters for global vs. thread local cache are needed since there will be delay when sampling those (sum might not match the pool size).

Reply:
Fixed in v2